### DISCUSSION TITLE
Does anyone know how to add the forgotten combo.png

### DISCUSSION THREAD

**Discussion by Theoyeah** [DEMOLITIONDON96/Demolition-Engine#40] [Source Q&A]
in 0.5.2h (:

**Comment by ThatOneSillyGuyDon** [comment]
I think it’s getting added back in the official game, so, maybe we can use that code if they release it, or we can use the unused code of it in the engine rn

**Reply by Theoyeah** [reply]
yes
so how do we do it ?

**Reply by Theoyeah** [reply]
?

**Reply by RealDealJinx** [reply]
it is but it will be like 10 combo or some shit

**Reply by Theoyeah** [reply]
ok

**Comment by Theoyeah** [comment]
hhhhheeeellllpppp i really need howwwww someone tell meeee
